### ISSUE TITLE
http://demo.hotspot.kiwix.org should redirect to https://demo.hotspot.kiwix.org

### ISSUE THREAD

**Issue by kelson42** [kiwix/k8s#183]
The reasons are IMHO:
* No reason to keep here `http://`
* `http://` should be avoided if possible for many good reasons, in particular security one. SEO would be an other one
* Worse: if we have ZIMit1 ZIM files, then http://demo.hotspot.kiwix.org/ won't be able to deliver smoothly ([like with current version](http://kiwix.demo.hotspot.kiwix.org/viewer#openzim_grimgrains_en_all_/A/index.html)) which gives a bad impression about the product

**Comment by rgaudin** [comment]
I disagree. This is a replica of the hotspot product which works exactly like this: you access it via HTTP and there is no redirect to HTTPs. I don't see what SEO has to do with this given we want to exclude it from search results. As for the Zimit1 scenario, it has the advantage of showing how it works on the hotspot (kind of the point of the demo!). The “bad impression” is due to the ZIM itself, not the hotspot.

We can obviously do it technically but:
- I like that it is similar to the actual hotspot (still waiting on a written description of this tool's objective)
- I don't like having code in hotspot that is triggered conditionally for the demo
- I think it's not necessary. The only argument is that https would protect demo-users from MITM.

So please reconsider and let me know you final decision. Change would be in container-images so it will require recreating an image for it to apply anyway.

**Comment by kelson42** [comment]
@rgaudin If something does not work smoothly it should not be in the demo. Maybe we should just not put Zimit1 stuff in it as this is anyway not readable in the field?

**Comment by rgaudin** [comment]
that's my opinion as well

**Comment by benoit74** [comment]
Content fixed by @Popolechien to not include zimit1 stuff.

**Comment by Popolechien** [comment]
I've removed the zimit1 content but I am not sure it solved the initial `http`-> `https`issue

**Comment by benoit74** [comment]
It did not solved  `http` -> `https` issue because there is no issue, the demo is replicating exactly the hotspot behavior (where there is no redirection)

**Comment by Popolechien** [comment]
Yeah the issue was not really one (I'd side with @rgaudin on this) but I wanted to flag it nonetheless.

**Comment by rgaudin** [comment]
I've opened https://github.com/offspot/demo/issues/7 for the side effect that got @Popolechien  to open another issue.
